src/main/java/lotto/domain/number/LottoNumbersGenerator.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain.number;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbersGenerator {
+
+    private static final List<Integer> RANGED_LOTTO_NUMBERS =
+            IntStream.rangeClosed(LottoNumber.MIN_VALUE, LottoNumber.MAX_VALUE)
+                    .boxed()
+                    .collect(Collectors.toList());
```

Comment:
여기서 조금만 더 나가면
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS_POOL =
            IntStream.rangeClosed(LottoNumber.MIN_VALUE, LottoNumber.MAX_VALUE)
                    .boxed()
                    .map(LottoNumber::of)
                    .collect(Collectors.toList());
```
캐시해서 재사용 할 수 있고, 아래쪽에서 converToLottoNumbers 함수도 없앨수 있겠어요!

Reply:
오호 그렇네요!!ㅎㅎㅎ 👍